docker/Dockerfile.dev

```diff
@@ -0,0 +1,30 @@
+FROM ubuntu
+
+MAINTAINER perryism
+ 
+RUN apt-get update && \
+    apt-get -y install \
+	wget \
+	zip \
+	python-pip \
+	git \
+    && rm -rf /var/lib/apt/lists/* 
+
+ARG VERSION=0.8.8
+
+RUN wget https://github.com/airbnb/knowledge-repo/archive/v$VERSION.zip && \
+    unzip v$VERSION.zip -d /app
+
+WORKDIR /app/knowledge-repo-$VERSION
+
+COPY . /app/knowledge-repo-$VERSION
```

Comment:
Why do you copy the whole directory into `/app/knowledge-repo-$VERSION`?
Wouldn't it be enough to just copy `requirements.txt`?